lib/class-wp-rest-sidebars-controller.php

```diff
@@ -404,33 +249,30 @@ public function prepare_item_for_response( $raw_sidebar, $request ) {
 
 		$fields = $this->get_fields_for_response( $request );
 		if ( rest_is_field_included( 'widgets', $fields ) ) {
-			$sidebar['widgets'] = self::get_widgets( $sidebar['id'], $request );
+			$sidebars = wp_get_sidebars_widgets();
+			$widgets  = array_filter(
+				isset( $sidebars[ $sidebar['id'] ] ) ? $sidebars[ $sidebar['id'] ] : array(),
+				function ( $widget_id ) {
+					global $wp_registered_widgets;
+
+					return isset( $wp_registered_widgets[ $widget_id ] );
+				}
```

Comment:
```suggestion
				function ( $widget_id ) use ( $wp_registered_widgets ) {

					return isset( $wp_registered_widgets[ $widget_id ] );
				}
```

Would this be cleaner?